.changeset/hungry-cycles-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Hds::Pagination`
```

Comment:
Small, but usually we've written the changelogs as:
```suggestion
`Hds::Pagination` - Converted component to Typescript
```

(instead of 2 lines)